Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -77,18 +77,23 @@ theorem lintegral_edist_triangle {f g h : α → β} (hf : AEStronglyMeasurable
   refine lintegral_mono fun a => ?_
   apply edist_triangle_right
 
-theorem lintegral_nnnorm_zero : (∫⁻ _ : α, ‖(0 : β)‖₊ ∂μ) = 0 := by simp
+-- Yaël: Why do the following four lemmas even exist?
```

Comment:
I'm reviewing commit-by-commit: just flagging this so it gets taken care of before merging.